CHANGES.rst

```diff
@@ -1,6 +1,20 @@
 SimPhoNy Remote CHANGELOG
 =========================
 
+What's new in SimPhoNy Remote 0.2.0
+-----------------------------------
+
+Summary
+~~~~~~~
+
+- Introduced access control for images and users by means of a database.
+  Additionally, a CLI utility to modify the content of the database has
+  been provided (#8)
+- Added readthedocs documentation (#12)
+- Container URL now contains a base32 encoded unique identifier, 
+  instead of the docker container id. (#18)
+- Introduced authentication of the user for the application (#24)
+- Bugfixes (#14, #15)
```

Comment:
"Bugfixes (#14, #15)" -> 'Improved handling of failures in starting containers (#6, #14, #15)'